initial_bot/mentor_bot_test.py

```diff
@@ -73,6 +74,29 @@ async def on_member_join(member):
             f'Hi {member.name}, welcome to my Discord server!'
         )
 
+    # Save messages that were sent in private mentoring rooms
+    @bot.event
+    async def on_message(message):
+        '''        
+        private mentoring rooms will have some identifier in their names, so the bot can decide whether or not to save the message.
+        For now I am tentatively using "shlomi" as the identifier :-)
+        
+        '''
+        prvt_ment_channels_mark = 'shlomi'
+        channel = message.channel
+        author = message.author
+        if not author.bot and prvt_ment_channels_mark in channel.name:
+            with open(f"initial_bot/{channel.name}_messages.txt", 'a+') as f:
+                date = (message.created_at)
+                f.write(f'{date:%d/%m/%Y %H:%M}\n{author.name}\n')
+                f.write(message.content + "\n\n")
+            bot.dispatch('documentation', channel, author.name)
+    
+    @bot.event
+    async def on_documentation(channel, author):
```

Comment:
In the final version I don't think we'll need this since the bot will flood the chat as the conversation goes on, of course for testing its great

Reply:
Totally agree